Dockerfile.sdk

```diff
@@ -78,15 +79,15 @@ RUN apt-get update && \
             python3 \
             python3-pip \
             python3-dev \
+            python3-wheel \
+            python3-setuptools \
```

Comment:
Why we are adding setuptools package?

Reply:
It was removed from the pip install below and added here, cause of the new pip behaviour with system installations. No new install overall